index.js

```diff
@@ -22,3 +22,7 @@ if (typeof window !== 'undefined') {
 }
 
 require('raf/polyfill');
+
+global.requestIdleCallback = require('ric-shim');
```

Comment:
it'd be nice to file an upstream issue so that `require('ric-shim/shim')` would do this global installation for us.